### ISSUE TITLE
Not working with Angular 17

### ISSUE THREAD

**Issue by jisan3s** [dominique-mueller/angular-notifier#258]

**Comment by Identity-labs** [comment]
Abandoned project i think

**Comment by NazimMertBilgi** [comment]
Is there any progress on this issue?

**Comment by NazimMertBilgi** [comment]
@dominique-mueller

**Comment by KeatsPeeks** [comment]
Time to fork the entire project or switch to another lib.

**Comment by spock123** [comment]
Any good alternatives? 
ngx-toastr doesn't work with Zoneless change detection

**Comment by Ladzi031** [comment]
@spock123  try [](https://github.com/bersling/notify-angular) it worked on my previous project...

**Comment by Ladzi031** [comment]
@spock123  try this "notiflix"
 here is the link: https://notiflix.github.io/
it worked on my previous project...

**Comment by Gramli** [comment]
For those still interested, I created a fork and released versions for Angular 17 and 18. You can find the repository here: https://github.com/Gramli/angular-notifier.

Both versions are published on npm: https://www.npmjs.com/package/gramli-angular-notifier.

**Comment by ethan-far** [comment]
@Gramli could you open a PR with your changes?

**Comment by Gramli** [comment]
> @Gramli could you open a PR with your changes?

I think that this repository is abandoned, nobody will merge changes.

If you wnat to check all changes: [Compare-Develop](https://github.com/dominique-mueller/angular-notifier/compare/develop...Gramli:angular-notifier:develop) or there is last [PR for Angular 18 update](https://github.com/Gramli/angular-notifier/pull/1)

**Comment by ethan-far** [comment]
@Gramli Thanks, I'll use yours. Are you planning on maintaining it? I'd be happy to contribute as well if needed

**Comment by Gramli** [comment]
> @Gramli Thanks, I'll use yours. Are you planning on maintaining it? I'd be happy to contribute as well if needed

Yes, I am planning to maintain the repository. Feel free to contribute.
